armada_command/command_run.py

```diff
@@ -120,64 +160,73 @@ def command_run(args):
             print('Image {image} not found. Aborting.'.format(**locals()))
             sys.exit(1)
```

Comment:
We should raise exception instead and have one place to handle errors, which is: https://github.com/armadaplatform/armada/blob/766ea71c6cc65d291c95f5a48807c414842718c6/armada_command/armada.py#L154